test/trento/clusters_test.exs

```diff
@@ -46,6 +46,104 @@ defmodule Trento.ClustersTest do
       assert :ok = Clusters.request_checks_execution(cluster_id)
     end
 
+    test "should start a checks execution on demand for ascs_ers clusters with a resource managed filesystem type" do
```

Comment:
Maybe we could group these ascs/ers related tests in a more specific `ASCS/ERS Cluster Checks Execution`.